src/core/bpf/allow_bind/allow-bind.bpf.c

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: GPL */
+
+/* <linux/types.h> must precede <bpf/bpf_helpers.h> due to integer types
+ * in bpf helpers signatures.
```

Comment:
might be something to fix in the kernel headers?

Reply:
it's by design, actually. bpf_helpers.h can be used with linux/types.h and other system-level headers, as well as with a single vmlinux.h auto-generated header, which includes all the kernel types (down to u64 typedefs and so on), so bpf_helpers.h avoids making any assumptions and requirements about which headers should be included.